onnxruntime/python/tools/quantization/onnx_model.py

```diff
@@ -369,3 +368,93 @@ def topological_sort(self):
         assert end == len(self.graph().node), "Graph is not a DAG"
         self.graph().ClearField("node")
         self.graph().node.extend(sorted_nodes)
+
+    def clean_initializers(self):
+        return ONNXModel.clean_initializers_helper(self.graph(), self.model)
+
+    @staticmethod
+    def clean_initializers_helper(graph, model):
+        """
+        Clean unused initializers including which is caused by quantizing the model.
+            return cleaned graph, and list of tensor names from this graph and all its subgraphes
+            that can not be found in this graph and its subgraphes
+        """
```

Comment:
```suggestion
        """Clean unused initializers including which is caused by quantizing the model.
        
        Returns:
            A cleaned graph, and a list of tensor names from this graph and all its subgraphs
            that cannot be found in this graph and its subgraphs
        """
```

nit: formatting, wording